frontend/server/src/Controllers/School.php

```diff
@@ -270,33 +270,79 @@ public static function getTopSchoolsOfTheMonth(
      *
      * @omegaup-request-param int $length
      * @omegaup-request-param int $page
+     * @omegaup-request-param string $filter
```

Comment:
This field is optional, so you could use:
```suggestion
     * @omegaup-request-param null|string $filter
```